setup.py

```diff
@@ -34,7 +34,9 @@
     ],
     "tests": tests_require,
     "jq": [
-        "jq==1.4.1",
+        "jq==0.1.7; platform_machine == 'x86_64'",
```

Comment:
Maybe let's add a small comment here? Even just referencing the GitHub issue would be enough